lib/vector-common/src/datetime.rs

```diff
@@ -114,10 +114,7 @@ impl Configurable for TimeZone {
     fn generate_schema(gen: &mut SchemaGenerator) -> Result<SchemaObject, GenerateError> {
         let mut local_schema = generate_const_string_schema("local".to_string());
         let mut local_metadata = Metadata::<()>::with_description("System local timezone.");
-        local_metadata.add_custom_attribute(CustomAttribute::KeyValue {
-            key: "logical_name".to_string(),
-            value: "Local".to_string(),
-        });
+        local_metadata.add_custom_attribute(CustomAttribute::kv("logical_name", "Local"));
```

Comment:
I had to clean this up because `value` is now `serde_json::Value` instead of `String`, so this code was breaking. The `kv` helper method has all the helpful generic bounds and what not to make it easier to construct custom attributes directly in code.